app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,33 @@
+<div class='section sign'>
+  <h2 class='signup-head'>Sign Up</h2>
+
+  <%= simple_form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+    <%= f.error_notification %>
+
+    <div class="form-inputs signin">
+      <%= f.input :name,
+                  required: true,
+                  autofocus: true,
+                  input_html: { autocomplete: "name" }%>
+      <%= f.input :email,
+                  required: true,
+                  autofocus: true,
+                  input_html: { autocomplete: "email" }%>
+      <%= f.input :password,
+                  required: true,
+                  hint: ("#{@minimum_password_length} characters minimum" if @minimum_password_length),
+                  input_html: { autocomplete: "new-password" } %>
+      <%= f.input :password_confirmation,
+                  required: true,
+                  input_html: { autocomplete: "new-password" } %>
+    </div>
+
+    <div class="form-actions">
+          <%= f.button :submit, class: 'button is-dark signup_btn' %>
+
+    </div>
+  <% end %>
+
+
+<%= render "devise/shared/links" %>
+</div>
```

Comment:
- [ ] Your app working fine but you should stick to the design as required font, colors, layout and space elements as in the requirements 
- Screenshot of the requirements 
<img width="522" alt="Screenshot 2022-11-17 at 13 13 06" src="https://user-images.githubusercontent.com/61790587/202443960-d4cd1cc0-74fe-44fd-bd6e-b25dec798ba5.png">

- Yours 
<img width="372" alt="Screenshot 2022-11-17 at 13 12 07" src="https://user-images.githubusercontent.com/61790587/202443653-e043a8d1-8816-4c83-b9f0-95e8e8931190.png">
- Design 
<img width="276" alt="Screenshot 2022-11-17 at 13 13 48" src="https://user-images.githubusercontent.com/61790587/202443675-89184124-5e1b-43b3-a92d-03b2a65dfeef.png">

Reply:
I have made this as close as possible to the design. I hope you understand that I cannot change absolutely everything as there are some things in this design that are not part of my project such as the picture. Also the log in and sign in pages use devise, so I have to customize my app in that regard.